$OpenBSD: patch-lenses_shellvars_aug,v 1.7 2014/02/06 10:21:09 jasper Exp $

- adduser.conf is not a shell script, so don't try to parse it as such.

--- lenses/shellvars.aug.orig	Mon Jan 27 11:36:08 2014
+++ lenses/shellvars.aug	Thu Feb  6 11:15:37 2014
@@ -210,7 +210,6 @@ module Shellvars =
                      . incl "/etc/environment"
                      . incl "/etc/firewalld/firewalld.conf"
                      . incl "/etc/blkid.conf"
-                     . incl "/etc/adduser.conf"
                      . incl "/etc/cowpoke.conf"
                      . incl "/etc/cvs-cron.conf"
                      . incl "/etc/cvs-pserver.conf"
